Jenkinsfile.conda

```diff
@@ -3,7 +3,7 @@ pipeline {
     environment {
         package_name = "athexapod"
         dockerImageName = "lsstts/conda_package_builder:latest"
-        container_name = "salobj_${BUILD_ID}_${JENKINS_NODE_COOKIE}"
+        container_name = "athexapod_conda_${BUILD_ID}_${JENKINS_NODE_COOKIE}"
```

Comment:
To keep the Jenkinsfile generic, I would recommend not using the package name on places like this. I would just use 

```
container_name = "conda_${BUILD_ID}_${JENKINS_NODE_COOKIE}"
```